app/services/logger.js

```diff
@@ -0,0 +1,32 @@
+const bunyan = require('bunyan');
+const PrettyStream = require('bunyan-prettystream');
+const { get } = require('lodash');
+
+const Logger = require('./base_logger');
+
+const config = require('../config').get('logger');
+
+const env = process.env.NODE_ENV || 'dev';
+
+const loggerOptions = {
+  name: config.name || 'event-emitter',
+  streams: [
+    { level: 'error', stream: process.stderr },
+    ...get(config, 'streams', []),
+  ],
+  serializers: bunyan.stdSerializers,
+};
+
+if (env === 'dev') {
+  const prettyStream = new PrettyStream();
+  prettyStream.pipe(process.stdout);
+
+  loggerOptions.streams.push({
+    level: 'info',
+    stream: prettyStream,
+  });
+}
+
+const rootLogger = new Logger(bunyan.createLogger(loggerOptions));
```

Comment:
suppose that we deleted the folder for logging or we haven't got her, do we need to check this case for a error? or logger instance create a new folder for logs, don't it?

Reply:
![image](https://user-images.githubusercontent.com/9417851/38237373-bd7c24d8-372f-11e8-8822-b32b63ebc9e3.png)
In this case, child logger is created with the specific name and when writing f.e `log.info` in the console we can see this `component` field and understand were this log was emitted

Reply:
I checked this case and error has occurred

Reply:
You can check it. [Link](https://github.com/inst-event-emitter/eventemitter-backend/pull/7/commits/ad112cb8c1d9c64199e855f153ea3258174bfc77#diff-586cf549a824167942b3b214a6303c48R29)